configs/cascade_mask_rcnn_r50_c4_1x.py

```diff
@@ -0,0 +1,229 @@
+# model settings
+model = dict(
+    type='CascadeRCNN',
+    num_stages=3,
+    pretrained='modelzoo://resnet50',
+    backbone=dict(
+        type='ResNet',
+        depth=50,
+        num_stages=3,
+        strides=(1, 2, 2),
+        dilations=(1, 1, 1),
+        out_indices=(2, ),
+        frozen_stages=1,
+        normalize=dict(type='BN', frozen=True),
+        norm_eval=True,
+        style='pytorch'),
```

Comment:
We can use caffe-style for C4 models by default, since the performance gap is significant.